NSS-Day-Cohort-53/giffygram-spazzy-kermits
# messages sent to database when button is clicked

#### Changes Made
1. Added messageCreate.js to messages folder
1. modified messageCreate.js and messageList.js
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-sendMessage
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > button to send message now adds message to database, event listeners for rendering html in message list using a custom event
    > Example: When user hits send message button it should go to the recipient selected
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
